e2e/interchaintestv8/relayer/config.go

```diff
@@ -0,0 +1,117 @@
+package relayer
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/testvalues"
+)
+
+// SP1ProverType represents the type of SP1 prover
+type SP1ProverType string
+
+const (
+	ModuleCosmosToCosmos = "cosmos_to_cosmos"
+	ModuleCosmosToEth    = "cosmos_to_eth"
+	ModuleEthToCosmos    = "eth_to_cosmos"
+)
+
+// Config represents the relayer configuration structure and serves as template data
+type Config struct {
```

Comment:
Open to suggestions, but got a bit tired of having multiple template files to keep in sync. Just tried to move things around a bit without disturbing too much in the test files.